tests/python/test_mset.py

```
@@ -0,0 +1,383 @@
+        
+        # 空字典应该返回 True（Redis 行为）
+        result = r.mset({})
+        assert result == True
+
+
+# ============================================================================
+# 独立运行支持（兼容旧版脚本）
+# ============================================================================
+
+def run_standalone_tests():
+    """独立运行所有测试（不使用 pytest）"""
```

Ruff (0.14.1)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)